The compress test fails when running the entire test suite, however does not fail when running
the test standalone.  Adding 'FLUSH PRIVELEGES' seems to help fix the error, though not sure
why.

http://bugs.mysql.com/bug.php?id=28416
 

--- mysql-5.0.41/mysql-test/t/compress.test.timeout	2007-05-14 08:06:19.000000000 -0500
+++ mysql-5.0.41/mysql-test/t/compress.test	2007-05-14 08:08:19.000000000 -0500
@@ -8,6 +8,9 @@
 
 connect (comp_con,localhost,root,,,,,COMPRESS);
 
+# Redhat build fix
+FLUSH PRIVILEGES;
+
 # Check compression turned on
 SHOW STATUS LIKE 'Compression';
 
--- mysql-5.0.41/mysql-test/r/compress.result.timeout	2007-05-14 08:09:52.000000000 -0500
+++ mysql-5.0.41/mysql-test/r/compress.result	2007-05-14 08:09:00.000000000 -0500
@@ -1,3 +1,4 @@
+FLUSH PRIVILEGES;
 SHOW STATUS LIKE 'Compression';
 Variable_name	Value
 Compression	ON
